src/config-service.ts

```diff
@@ -21,14 +22,78 @@ export interface IConfigService {
   ): Promise<ISyncOptions>
 }
 
-const explorer = cosmiconfig('typesync')
+async function loadYaml(_filepath: string, content: string): Promise<unknown> {
+  return await yaml.parse(content)
+}
+
+// From
+function searchPlaces(moduleName: string) {
```

Comment:
lilconfig doesn't support as many locations as cosmiconfig by default (notably, `.rc` with no extension.

Reply:
(And I fixed that comment)